src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,268 @@
+
+    ```
+    HTTP_SPAN_LIMIT: 750
+    DB_SPAN_LIMIT: 250
+    COLLECT_INTERVAL_SEC: 15
+    ```
+
+    You can find the default settings for these parameters and others in the [newrelic-pixie-integration](https://github.com/newrelic/newrelic-pixie-integration) Github repo.
+
+    </Collapser>
+</CollapserGroup>
+
+
+## Reduce data ingested by changing the scrape interval [#scrape-interval]
+
+The New Relic Kubernetes integration allows you to change the scrape interval when metrics are gathered from the cluster. This lets you choose the right balance between data resolution and usage. We suggest choosing a scrape interval between 15 and 30 seconds for the best experience.
+
+<Callout variant="tip">
+    The `lowDataMode` parameter already sets the scrape interval to 30 seconds.
+</Callout>
+
+
+The modification of the scrape interval will depend on the method used to install Kubernetes:
+
+    * **Guided install** (New Relic CLI, Helm, Manifest): You can't modify this value following our guided install flow.
+
+    * **Helm**: Set the scrape interval in the [`nri-bundle` chart](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) to the value you want.
+
+    * **Manifest**: Set the scrape interval in the Manifest configuration yaml file to the value you want. If you followed our [installation instructions](/install/kubernetes/?dropdown1=manifest), the file was called `newrelic-manifest.yaml`.
+
+If you want to change the scrape interval, just add the interval value under the `newrelic-infrastructure` section. Take a look at this example of the `values-newrelic.yaml` file to see how it looks for Helm:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings
+  common:
+    config:
+      interval: 25s
 ```
-- kube_
-- container_
-- machine_
-- cadvisor_
+
+<Callout variant="important">
+    Note that if you're using the [cluster explorer](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#cluster-explorer-use) instead of the [Kubernetes navigator](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#navigator-preview), you're not allowed to set `interval` to values greater than `40s`.
+</Callout>
+
+
+## Reduce data ingested by filtering namespaces [#filter-namespace]
+
+You can label namespaces to filter which ones the Kubernetes integration scrapes. All namespaces scrape by default.
+
+We use the `namespaceSelector` in the same way Kubernetes does. To include only namespaces matching a label, just change the `namespaceSelector`. Add the following to your `values-newrelic.yaml` file, under the `newrelic-infrastructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        key1 : "value1"
 ```
 
```

Comment:
I would add an h3 heading here like this: ### Example: Include namespaces [#include]`

Reply:
Done.